docs/tools/CLI/cli-compile.md

```diff
@@ -75,65 +88,73 @@ To rebuild the project, use the `-c/--clean` argument:
+By default, Mbed CLI includes the program's root directory and all its subdirectories as the source directory. To control which directories are used, use the `--source` argument. You can supply multiple `--source` arguments to include multiple directories:
 
 ```
 $ mbed compile --source ./src --source ./lib
 ```
 
 ### Build directory
 
-The default build directory is in the root of your project in a directory called `BUILD`. You can set the build directory `--build` argument:
+The default build directory is in the root of your project, and is called `BUILD`. You can set the build directory with the `--build` argument:
 
 ```
 $ mbed compile --build my_build
 ```
 
-### Build profiles
-
-Build profiles are used to control what arguments are passed to the compiler. For more information about build profiles, please see the [build profile documentation](../tools/build-profiles.html).
+### Build a library
+<!--this feels like it should be "building", especially since we have "build directory"; "building" makes it clearer that this is a verb, whereas the previous one was a noun-->
 
-The default build profile is set to `develop`, which enables debug logging. There is also a `debug` profile (for enabling debug symbols) and a `release` profile (for disabling debug logging, saving program size). This is controlled with the `--profile` argument:
+To build a static library instead of a linked executable, use the `--library` argument:
 
 ```
-$ mbed compile --profile debug
+$ mbed compile --library
 ```
 
-### Build a library
+To suppress the creation of the `.a/.ar` archive (and instead leave the `.o` object files), use both the `--library` and `--no-archive` arguments:
+
+<!--example please-->
+## Build profiles
+
+Build profiles control which arguments are passed to the compiler. For more information about build profiles, please see the [build profile documentation](../tools/build-profiles.html).
+
+The default build profile is `develop`, which enables debug logging. There is also a `debug` profile, which enables debug symbols, and a `release` profile that disables debug logging (to reduce the program size).
 
-If you wish to build a static library instead of a linked executable, you can use the `--library` argument:
+To select a profile, use the `--profile` argument:
 
 ```
-$ mbed compile --library
+$ mbed compile --profile debug
 ```
 
-To suppress the creation of the `.a/.ar` archive (and instead leave the `.o` object files), use the `--no-archive` argument in addition to the `--library` argument. 
-
 ## Configuration system
 
-You can use `mbed compile --config` to view the configuration (use the `-v` argument to display more information):
+The Mbed OS configuration system customises compile time configuration parameters. For more info, [see the full configuration system documentation](./reference/configuration.html).
```

Comment:
We use American English in our docs right? If so, `customises` should be `customizes` (embrace the 💤).